SparseGrids/Benchmarks/benchFFT.hpp

```diff
@@ -0,0 +1,59 @@
+#ifndef _TASMANIAN_BENCHMARK_FFT_HPP
+#define _TASMANIAN_BENCHMARK_FFT_HPP
+
+#include "benchCommon.hpp"
+
+bool benchmark_fft(std::deque<std::string> &args){
+    if (args.size() < 5) return false;
+
+    // report the test parameters to reference later
+    cout << "fourier";
+    for(auto &s : args) cout << " " << s;
+    cout << endl;
+
+    auto grid_family = GridFamily::fourier;
+
+    int num_dimensions, num_depth, iteratons, num_jumps;
+    TypeDepth dtype;
+
+    auto riter = readEntries(args.begin(), num_dimensions, num_depth, dtype, iteratons, num_jumps);
+
+    auto extra = extractWeightsLimits(grid_family, num_dimensions, dtype, riter, args.end());
+
+    int num_outputs = 1;
+    auto make_grid = getLambdaMakeGrid(grid_family, num_dimensions, num_outputs, num_depth, dtype, rule_fourier, 0, extra);
```

Comment:
* pass the order to the lambda
* pass the rule to the lambda, the function will ignore the rule when using Fourier grids but it is needed for other cases